xdsl/ir.py

```diff
@@ -972,6 +972,16 @@ def _attach_op(self, operation: Operation) -> None:
             )
         operation.parent = self
 
+    @property
+    def first_op(self) -> Operation | None:
+        """The first operation in `self`"""
+        return self.ops[0] if len(self.ops) else None
+
+    @property
+    def last_op(self) -> Operation | None:
+        """The last operation in `self`"""
```

Comment:
```suggestion
        """The last operation in the block."""
```
Maybe?

Reply:
How about `"""The last operation in this block."""`?